HHS/OPRE-OPS
# update route for Agreements Review page

## What changed

update route for Agreements Review page

## Issue

#1614 

## How to test

1. Goto agreements list
2. click airplane icon
3. see AgreementReview page

## Definition of Done Checklist
